be/src/storage/lake/rowset.h

```diff
@@ -114,13 +116,15 @@ class Rowset : public BaseRowset {
     TabletSchemaPtr _tablet_schema;
     TabletMetadataPtr _tablet_metadata;
     std::vector<SegmentSharedPtr> _segments;
+    int64_t _version;
```

Comment:
no need to add `_version`, we already have version in `_metadata`.

Reply:
done